diff --git a/vertica_python/vertica/column.py b/vertica_python/vertica/column.py
index 75d87a2..9eeef08 100644
--- a/vertica_python/vertica/column.py
+++ b/vertica_python/vertica/column.py
@@ -5,6 +5,11 @@ from collections import namedtuple
 from decimal import Decimal
 from datetime import date
 from datetime import datetime
+from pkg_resources import require,VersionConflict
+try:
+	require("python_dateutil>=1.5")
+except VersionConflict:
+	pass
 from dateutil import parser
 
 import pytz
