diff --git a/vertica_python/vertica/connection.py b/vertica_python/vertica/connection.py
index 2814b25..35e1b32 100644
--- a/vertica_python/vertica/connection.py
+++ b/vertica_python/vertica/connection.py
@@ -131,10 +131,10 @@ class Connection(object):
             raise TypeError("invalid message: ({0})".format(message))
 
         if getattr(self, 'debug', False):
-            print "=> {0}".format(message)
+            print("=> {0}".format(message))
         try:
             self._socket().sendall(message.to_bytes())
-        except Exception, e:
+        except Exception as e:
             self.close_socket()
             raise errors.ConnectionError(e.message)
 
@@ -164,7 +164,7 @@ class Connection(object):
                     raise errors.MessageError("Bad message size: {0}".format(size))
                 message = BackendMessage.factory(type, self.read_bytes(size - 4))
                 if getattr(self, 'debug', False):
-                    print "<= {0}".format(message)
+                    print("<= {0}".format(message))
                 return message
             else:
                 self.close()
