diff --git a/vertica_python/__init__.py b/vertica_python/__init__.py
index 487f376..ea24bf3 100644
--- a/vertica_python/__init__.py
+++ b/vertica_python/__init__.py
@@ -24,7 +24,7 @@ from vertica_python.errors import TimedOutError, ConnectionError, SSLNotSupporte
 # Main module for this library.
 
 # The version number of this library.
-version_info = (0, 1, 9)
+version_info = (0, 2, 0)
 __version__ = '.'.join(map(str, version_info))
 
 __author__ = 'Uber Technologies, Inc'
